docs/developers/docker.rst

```diff
@@ -0,0 +1,82 @@
+.. _docker:
+
+Docker
+======
+
+You can install a development and test environment using docker.
+
+
+.. _docker#assumptions:
+
+
+Assumptions
+-----------
+
+- You have docker installed and working on your system
+- The user you are using has permissions to use docker
+
+
+.. _docker#virtualenv:
+
+Create a virtualenv (optional)
+------------------------------
+
+Installing with virtualenv keeps your host environment clean, and separates
+your development environment from the host.
+
+For example:
+
+.. code-block:: console
+
+  $ mkvirtualenv pootle
+  $ cd ~/virtualenvs/pootle
+
+
+.. _docker#clone:
+
+Clone the pootle repository
+---------------------------
+
+You most likely want to clone *your* fork of the pootle repository, so you
+can easily create Pull Requests for your changes.
```

Comment:
I would rephrase to enforce developers using their fork. But for people doing a test run having a fork in place is not mandatory.